lib/analytics.js

```diff
@@ -296,6 +316,7 @@ Analytics.prototype.track = function(event, properties, options, fn) {
   // figure out if the event is archived.
   var plan = this.options.plan || {};
   var events = plan.track || {};
+  var planIntegrationOptions = {};
```

Comment:
The changes around `planIntegrationOptions` are not gated here. It does seem ok since it's gated in `_mergeInitializeAndPlanIntegrations` instead - but @Peripheral1994 could you take a second look to confirm expected behaviour as well?